### PR TITLE
Add recent papers in documentation

### PR DESCRIPTION
**Pull request by RemiLehe**

### CODE REVIEW

**Comment by EZoni** [comment]
Shall we add/update https://journals.aps.org/pre/abstract/10.1103/PhysRevE.110.025206, too?

**Comment by RemiLehe** [comment]
@EZoni absolutely, yes. I will add this.

File: Docs/source/acknowledge_us.rst [hunk 1]
```diff
@@ -53,6 +53,11 @@ Prior WarpX references
 
 If your project uses a specific algorithm or component, please consider citing the respective publications in addition.
 
+- Shapoval O, Zoni Z, Lehe R, Thévenet M, and Vay J-L
```
Comment (EZoni) [inline Docs/source/acknowledge_us.rst]: Just a typo:
```suggestion
- Shapoval O, Zoni E, Lehe R, Thévenet M, and Vay J-L
```

File: Docs/source/highlights.rst [hunk 2]
```diff
@@ -185,3 +195,16 @@ Nuclear Fusion - Magnetically Confined Plasmas
...
+
+#. Xie L., Luo X., Zhou Z. and Zhao Y.,
+   **Effect of plasma initialization on 3D PIC simulation of Hall thruster azimuthal instability**.
+   Physica Scripta, **99**, 095602, 2024.
+   `DOI:10.1088/1402-4896/ad69e5 <https://doi.org/10.1088/1402-4896/ad69e5>`__
+
+#. Marks T. A. and Gorodetsky A. A.,
+   **Hall thruster simulations in WarpX**.
+   38th International Electric Propulsion Conference, Toulouse, France, 2024.
+   `DOI:10.7302/234915 <https://doi.org/10.7302/23491>`__
```
Comment (ax3l) [inline Docs/source/highlights.rst:210]:
```suggestion
   `DOI:10.7302/234915 <https://doi.org/10.7302/23491>`__

#. Boeuf J-P, Smolyakov A.
   **Physics and instabilities of low-temperature E × B plasmas for spacecraft propulsion and other applications**.
   Physics of Plasmas **30**, 050901, 2023.
   `DOI:10.1063/5.0145536 <https://doi.org/10.1063/5.0145536>`__
```

File: Docs/source/acknowledge_us.rst [hunk 3]
```diff
@@ -53,6 +53,11 @@ Prior WarpX references
 
 If your project uses a specific algorithm or component, please consider citing the respective publications in addition.
 
+- Shapoval O, Zoni E, Lehe R, Thévenet M, and Vay J-L
```
Comment (ax3l) [inline Docs/source/acknowledge_us.rst]:
```suggestion
- Shapoval O, Zoni E, Lehe R, Thévenet M, and Vay J-L.
```

File: Docs/source/highlights.rst [hunk 4]
```diff
@@ -85,6 +85,11 @@ Laser-Plasma Interaction
 
 Scientific works in laser-ion acceleration and laser-matter interaction.
 
+#. Garten M, Bulanov S S, Hakimi S, Obst-Huebl L, Mitchell C E, Schroeder C B, Esarey E, Geddes C G R, Vay J-L, and Huebl A.
+   **Laser-plasma ion beam booster based on hollow-channel magnetic vortex acceleration**.
+   Phys. Rev. Research **6**, 033148, 2024.
```
Comment (ax3l) [inline Docs/source/highlights.rst]:
```suggestion
   Physical Review Research **6**, 033148, 2024.
```

File: Docs/source/highlights.rst [hunk 5]
```diff
@@ -85,6 +85,11 @@ Laser-Plasma Interaction
 
 Scientific works in laser-ion acceleration and laser-matter interaction.
 
+#. Garten M, Bulanov S S, Hakimi S, Obst-Huebl L, Mitchell C E, Schroeder C B, Esarey E, Geddes C G R, Vay J-L, and Huebl A.
```
Comment (ax3l) [inline Docs/source/highlights.rst]:
```suggestion
#. Garten M, Bulanov S S, Hakimi S, Obst-Huebl L, Mitchell C E, Schroeder C B, Esarey E, Geddes C G R, Vay J-L, Huebl A.
```

**Review by ax3l — APPROVED** [review]
Thank you! :)

File: Docs/source/highlights.rst [hunk 6]
```diff
@@ -185,3 +195,21 @@ Nuclear Fusion - Magnetically Confined Plasmas
...
+
+#. Marks T. A. and Gorodetsky A. A.,
+   **Hall thruster simulations in WarpX**.
+   38th International Electric Propulsion Conference, Toulouse, France, 2024.
+   `DOI:10.7302/234915 <https://doi.org/10.7302/23491>`__
+
+#. Boeuf J-P, Smolyakov A.
+   **Physics and instabilities of low-temperature E × B plasmas for spacecraft propulsion and other applications**.
+   Physics of Plasmas **30**, 050901, 2023.
+   `DOI:10.1063/5.0145536 <https://doi.org/10.1063/5.0145536>`__
```
Comment (RemiLehe) [inline Docs/source/highlights.rst]:
```suggestion
```

Comment (ax3l) [reply Docs/source/highlights.rst]: Correct, just aspirational in the end :+1:

**Pull request merged**